packages/webpack-config/package.json

```diff
@@ -40,7 +40,7 @@
     "url": "https://github.com/expo/expo-cli/issues"
   },
   "dependencies": {
-    "@babel/core": "7.9.0",
+    "@babel/core": "^7.16.0",
```

Comment:
Ensure this works with the version of babel-loader -> webpack@4

Reply:
I did not noticed any issues with Webpack 4 which is still used in this package. 

Do you have any exact test or check in mind which I should perform?